manager/src/main/java/com/redhat/service/bridge/manager/api/internal/ShardBridgesSyncAPI.java

```diff
@@ -67,7 +67,8 @@
     @PUT
     @Path("processors")
     public Response updateProcessorStatus(ProcessorDTO processorDTO) {
-        failIfNotAuthorized(jwt);
+        String shardId = customerIdResolver.resolveCustomerId(jwt);
```

Comment:
The naming is confusing, is it shard id or customer id?

Reply:
`customerIdResolver.resolveCustomerId` identifies the subject of the request from the `jwt` token. 

We might find a better name for this since as you pointed out the request might come from robot accounts and not from real customers

Reply:
Agreed. `IdentityResolveer`, `RequestIdentityResolver` as some initial suggestions 🤔